specific/cmo_sale/models/sale.py

```diff
@@ -93,11 +98,11 @@ def _compute_project_number(self):
 
     @api.model
     def _default_covenant(self):
-        convenants = self.env['sale.convenant.description'].search([
+        covenants = self.env['sale.covenant.description'].search([
```

Comment:
sure, covenant is always have only one.  i need it to be a default value. In the future if want to create new covenant let user set old covenant active = False then create new covenant.